sources/packages/web/src/services/AuthService.ts

```diff
@@ -93,6 +107,20 @@ export class AuthService {
       });
 
       if (this.keycloak.authenticated) {
+        // In case of user closed browser without logout.
+        window.addEventListener("beforeunload", this.logUserClosedBrowser);
+        if (AuditService.hasUserClosedBrowser()) {
+          // In case of user reopened browser with an active session.
+          await ApiClient.AuditApi.audit({ event: AuditEvent.BrowserReopened });
+          AuditService.resetUserClosedBrowser();
+        }
+        if (AuditService.wasUserLoginTriggered()) {
+          // Call audit api to log user logon.
+          await ApiClient.AuditApi.audit({ event: AuditEvent.LoggedIn });
+          AuditService.resetLoginTriggered();
+          AuditService.resetUserClosedBrowser();
```

Comment:
Can `AuditService.resetUserClosedBrowser();` be removed from lines 115 and 121 and just be called after the `if` conditions?
I would say that even the `AuditService.resetLoginTriggered();` can be moved outside the if condition.